en/docs/learn/consume-api/account-management-dev-portal/recover-dev-portal-password.md

```diff
@@ -0,0 +1,18 @@
+# Recover Password for the Developer Portal
+
+!!! note
+    The password recovery feature does not work out-of-the-box because an email server needs to be configured to be able to send the password recovery email. For more information, see Enable Password Recovery for the Developer Portal.
+
+1.  Click **Forgot Password** on the Sign In page of the Developer Portal and request a password change.
+
+    <img src="{{base_path}}/assets/img/administer/product-security/identity-management-for-the-api-dev-portal/forgot-password.png" alt="Forgot password link in sign in page" width="400px"/>
+
+3.  Enter the username you are trying to recover the password of and click **Submit**.
+
+    <img src="{{base_path}}/assets/img/administer/product-security/identity-management-for-the-api-dev-portal/password-recovery-form.png" alt="Password recovery page" width="400px"/>
+
+4.  You will receive an email with instructions to reset your password. 
+
+     <img src="{{base_path}}/assets/img/administer/product-security/identity-management-for-the-api-dev-portal/password-recovery-email-sent.png" alt="Password recovery email confirmation" width="450px"/>
+
+     The password recovery email is sent to the email address that is stored during the Developer Portal user sign up process.
```

Comment:
that is stored - > that is provided?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1555/commits/dcc2e157314cf300788569c01c2ad89f0877c4b3